packages/ui/src/zuri-ui.tsx

```diff
@@ -2,3 +2,9 @@ import "./global.css?modules=false";
 export * from "./Button";
 export * from "./config";
 export * from "./Layout";
+
+// onboarding pages
+export * from "./onboarding/Login/Layout";
+export * from "./onboarding/Login/pages/LoginPage";
```

Comment:
create these pages in the core app

it is not necessary to extract the whole page into the ui library

this will only make the ui library cluttered

a good rule of thumb is to ask yourself whether anyone would use that component (think about the plugins)